schedule.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+from os.path import abspath, dirname, join, isfile
+
+import sys
+from crontab import CronTab
+
+from settings import log_dir
+
+tab_path = "/etc/cron.d/bb8"
+
+
+def clear_existing(cron, backup_script):
+    existing = cron.find_command(backup_script)
+    for job in existing:
+        cron.remove(job)
+
+
+def add_job(cron, backup_script, test_job):
+    job = cron.new(command=backup_script, comment="BB8 backup", user="bb8")
```

Comment:
Should the user be configurable, here and in the `tab_path`? Or should our setup script actually create the bb8 user so we know that they must exist here.

Reply:
I think the user should be configurable because there are some other places (in the code used #1) where it would be useful.  If you have a concept of bb8 user then it's easy to run as either bb8 or as root because you can change user more easily.  It also helps with things like the interpretation of `~` etc because they all become `~<user>`

Reply:
Hmm, good question. I feel like it would be simpler if we always ran things as `bb8`. But the setup script probably can't be responsible for creating them: You should already be running as them before you run the setup script right? So perhaps we just need to have setup instructions in the README, like so:

```
git clone git@github.com:vimc/bb8
cd bb8
adduser bb8
./setup.sh
```

Reply:
Well unless you have an install script that you run as root, which creates the bb8 user and then acts as the bb8 user to run the setup script?

Reply:
This is also how we set up things during provisioning most places - write the script that runs as root and then change to the appropriate user